app/api/currentuser/get_group_memberships_history.go

```diff
@@ -8,6 +8,51 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 )
 
+// swagger:operation GET /current-user/group-memberships-history groups users groupsMembershipHistory
+// ---
+// summary: Display a history of invitations/requests for the current user
```

Comment:
Display -> return

Reply:
Or just "Get a history ..." as display is already displayed at the next line.

Reply:
fixed